tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "rootDirs": ["assets"],
+    "baseUrl": "assets",
+    "jsx": "react"
+  },
+  "exclude": ["node_modules"],
+  "include": [
+    "./assets/**/*"
+  ]
+}
```

Comment:
Looks like a sensible set of settings. At my company we use [these defaults](https://github.com/OpenConext/Stepup-Webauthn/blob/develop/tsconfig.json). Comparing the two, we have some more strict settings explicitly enabled. Other than that, your setup looks good to go!

Reply:
Also, adding Jest to test the JS code might be a good idea, for now, the low complexity might not warrant this, but if complexity is increased this might become a good idea.

Reply:
I use this because is the default with create-react-scripts. I have no problem to change it.